<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="EraserBtn">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/eraser_2891824.png</normaloff>C:/Users/Jay/Downloads/eraser_2891824.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="SquareBtn">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/menu_15990255.png</normaloff>C:/Users/Jay/Downloads/menu_15990255.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="CircleBtn">
    <property name="geometry">
     <rect>
      <x>220</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/stress-relieving_16679631.png</normaloff>C:/Users/Jay/Downloads/stress-relieving_16679631.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>40</width>
      <height>40</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="BrushBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/paint-brush_7605464.png</normaloff>C:/Users/Jay/Downloads/paint-brush_7605464.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="LineBtn">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/divide_9210852.png</normaloff>C:/Users/Jay/Downloads/divide_9210852.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UndoBtn">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/previous_4421621.png</normaloff>C:/Users/Jay/Downloads/previous_4421621.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>35</width>
      <height>35</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="PaintBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>130</y>
      <width>761</width>
      <height>431</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QPushButton" name="LeftArrowBtn">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/left-arrow_10068601.png</normaloff>C:/Users/Jay/Downloads/left-arrow_10068601.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="RightArrowBtn">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/right-arrow_10068204.png</normaloff>C:/Users/Jay/Downloads/right-arrow_10068204.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="UpArrowBtn">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/up-arrow_10067607.png</normaloff>C:/Users/Jay/Downloads/up-arrow_10067607.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DownArrowBtn">
    <property name="geometry">
     <rect>
      <x>640</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>C:/Users/Jay/Downloads/down-arrow_10068402.png</normaloff>C:/Users/Jay/Downloads/down-arrow_10068402.png</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>45</width>
      <height>45</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="RedoBtn">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>10</y>
      <width>61</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="SelectBtn">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Select</string>
    </property>
   </widget>
   <widget class="QPushButton" name="ColourBtn">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="text">
     <string>Colour</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
  </widget>
  <action name="actionNew">
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+X</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+V</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
